CHANGELOG.md

```diff
@@ -62,6 +62,15 @@ New Features
   truth information.  cf. demos 9 and 10. (#301, #691)
 - Added methods calculateHLR, calculateMomentRadius, and calculateFWHM to both
   GSObject and Image. (#308)
+- Added LookupTable2D to facilitate quick interpolation of two-dimensional
+  tabular data. (#465)
+- Added AtmosphericPhaseScreen, OpticalPhaseScreen, and PhaseScreenList used
+  to generate PSFs via Fourier optics. (#549)
+- Added PhaseScreenPSF to transform PhaseScreens into GSObjects.  (#549)
+- Added Atmosphere function to conveniently assemble a multi-layer atmosphere
+  PhaseScreenList. (#549)
+- Rewrote OpticalPSF to operate under the PhaseScreen framework to enable
```

Comment:
Given how often I've run into the "you can only do zernikes up to 11?" question, I think it would be nice to mention that the rewritten OpticalPSF can represent zernikes that go arbitrarily high.